lib/image.rb

```diff
@@ -67,8 +52,8 @@ def initialize(**attributes)
       attributes.map { |(k, v)| public_send("#{k}=", v) }
     end
 
-    def self.image(**kwargs)
-      new(file_nature: :image, **kwargs)
+    def nature
```

Comment:
Why don't we forego the constant and just return a symbol here? After all, Symbols are reused in Ruby. Besides, there is an `attr_accessor` which will work like this (if used at all):

```
image_info.nature = :powerpoint # will totally succeed
image_info.nature #=> :image
```

So we should have either an accessor, or a hardcoded reader - but not both.

Reply:
The attribute_accesor might a leftover from FileInformation, it shouldn't be there. Just a hardcoded reader is what we need.